spec/api/sul_bib/authorship_api_spec.rb

```diff
@@ -458,71 +423,51 @@ def check_response_error(code)
   # PATCH
 
   context 'PATCH /authorship' do
-    context 'success' do
-      let(:http_request) do
-        patch '/authorship', request_data.to_json, headers
-        expect(response.status).to eq(200)
-      end
+    let(:http_request) { patch '/authorship', request_data.to_json, headers }
 
+    context 'success' do
       # The POST specs use either sul_author_id or cap_profile_id for creating
       # new contributions, because  it can create new authors for
       # cap_profile_id.  Testing the author parameters is not required for POST
       # requests that update a contribution (where the author already exists).
       # For all the PATCH specs, the contribution already exists, so testing
       # different author params is not important.
+      after(:example) { expect(response.status).to eq(200) }
 
-      context 'to update all contribution attributes' do
-        let(:request_data) { update_authorship_for_pub_with_contributions }
-        it_behaves_like 'it updates an existing contribution'
-      end
+      it_behaves_like 'it updates an existing contribution'
```

Comment:
I seem to be missing what's happened to the request data in the "new" vs. "update" contexts.

Reply:
PATCH requests are by definition only updates.

Reply:
Ah, yes.  OK, good.